tests/unit/array/storage/qdrant/test_find.py

```diff
@@ -0,0 +1,34 @@
+from docarray import Document, DocumentArray
+import numpy as np
+
+
+def test_success_find_with_added_kwargs(start_storage, monkeypatch):
```

Comment:
You can parametrize the test with `search` params and expected, so u can test both cases